sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -164,10 +166,13 @@ public partial class SearchOptions
         public Azure.Search.Documents.Models.SearchMode? SearchMode { get { throw null; } set { } }
         public System.Collections.Generic.IList<string> Select { get { throw null; } }
         public string SemanticConfigurationName { get { throw null; } set { } }
+        public Azure.Search.Documents.Models.SemanticErrorHandling? SemanticErrorHandling { get { throw null; } set { } }
         public System.Collections.Generic.IList<string> SemanticFields { get { throw null; } }
+        public int? SemanticMaxWaitInMilliseconds { get { throw null; } set { } }
         public string SessionId { get { throw null; } set { } }
         public int? Size { get { throw null; } set { } }
         public int? Skip { get { throw null; } set { } }
+        public Azure.Search.Documents.Models.Vector Vector { get { throw null; } set { } }
```

Comment:
For now, we agreed to leave these matching the research literature for data scientists getting started.  We said we'd reevaluate based on customer feedback during previews and simultaneously do a lot with our docs to help folks tune the HNSW params.

There's this paradoxical behavior of the information retrieval research community to name things in ways that are absolutely unsearchable.  For example, in our first release I really, _really_ wanted to find friendlier names for things like [BM25](https://learn.microsoft.com/en-us/dotnet/api/azure.search.documents.indexes.models.bm25similarity?view=azure-dotnet) but it turns out that's what its called in all the textbooks, all the other search engine APIs, etc.

I feel much stronger about renaming `K` than `M` because while data scientists might be setting up the index, I think we'll have a broader set of users making queries.